README.md

```diff
@@ -1 +1,21 @@
-# Sport
\ No newline at end of file
+# Football_teams
+
+### About :
+
+our website allows user to search about footbal teams in specific country ,displays the teams in this country, and some details about it.
+
+### User journey:
+
+- As a user I should be able to enter a country name.
```

Comment:
You can simplify this to:

As a user, I will be able to:
- enter a country name
- click on search button and see the all teams in this country.
- click on team name and see more details about them.